chaosazure/__init__.py

```diff
@@ -50,6 +51,22 @@ def init_client(
         return client
 
 
+def init_website_management_client(
```

Comment:
Could we have a docstring here please, to explain its target?

I'm asking because now we have two ways to init a client and, not being an Azure user, I don't know when to use which one :)

Reply:
Hey @Lawouach _as far as I am able to tell_ you would need a separate client for every Azure service whose management client you wish to use. I came to this assumption by going to the [Azure Python SDK docs](https://docs.microsoft.com/en-us/python/api/overview/azure/?view=azure-python). Unless I'm mistaken, Azure Kubernetes Services, Virtual Machine Scale Sets and Virtual Machines are able use the Azure Compute management client to perform actions and probes to the underlying virtual machines. App Services (Azure Web Apps) is very different from virtual machines and must therefor use a different management client.

I believe this is the case for a lot of other Azure services. For instance let's say you want to target Azure Data Factory, which is Azure's cloud ETL solution, then you would need to use the DataFactoryManagementClient from  azure.mgmt.datafactory.DataFactoryManagementClient to perform management type actions.